src/main/java/com/nirima/jenkins/plugins/docker/builder/DockerBuilderNewTemplate.java

```diff
@@ -57,6 +57,7 @@
     public final String bindPorts;
     public final boolean bindAllPorts;
     public final boolean privileged;
+    public final boolean togglemode;
```

Comment:
Good idea, I think exclusive should be good for this. So if exclusive = false then it's Normal-Mode, if true then its Exclusive-Mode.